praw/models/reddit/subreddit.py

```diff
@@ -4034,6 +4049,34 @@ def upload_banner_hover_image(self, image_path: str):
         )
         self._update_structured_styles({image_type: image_url})
 
+    def upload_mobile_banner(self, image_path: str):
```

Comment:
Could you move this down (right before `upload_mobile_header` I believe) so they are sorted alphabetically please?